modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java

```diff
@@ -556,6 +563,12 @@ private JSONArray _getFiltersJSONArray(
 			(ObjectEntry objectEntry) -> {
 				Map<String, Object> properties = objectEntry.getProperties();
 
+				if (FeatureFlagManagerUtil.isEnabled("LPD-37531") &&
+					Boolean.FALSE.equals((Boolean)properties.get("active"))) {
+
+					return null;
+				}
+
```

Comment:
pls use the `predicate` argument in `_getSortedRelatedObjectEntries()` to filter inactive items out ([example](https://github.com/liferay/liferay-portal/blob/097e0e9d23f2d1e02aabaf7c95ab126c687af9c4/modules/apps/frontend-data-set/frontend-data-set-admin-web/src/main/java/com/liferay/frontend/data/set/admin/web/internal/fragment/renderer/FDSAdminFragmentRenderer.java#L343-L344))